### DISCUSSION TITLE
Some options in the BIOS are blocked

### DISCUSSION THREAD

**Discussion by felicefelice** [Atlas-OS/Atlas#1193] [Help & Support] [answered]
Hi,

I use a Lenovo ThinkPad X260.

In the BIOS, there are some options that I can't use, such as reseting the parameters. 

I wonder if it's something connected with using Atlas OS or if there si no link between the 2.

Thanks for your help.

Flc

**Comment by Xyueta** [comment, marked as answer]
Using Atlas does not mess up with your bios, Windows doesnt have much ability to manage them. However, it is possible they are locked by the manufacturer on purpose.

**Reply by felicefelice** [reply]
Thank you. Any idea how to unlock that?

**Reply by Xyueta** [reply]
> Thank you. Any idea how to unlock that?

Try using [SCEWIN](https://github.com/ab3lkaizen/SCEHUB) or [GRUB](https://github.com/BoringBoredom/UEFI-Editor#how-to-change-hidden-settings-without-flashing-a-modded-bios)
